template/.prettierrc.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Simply by having this file, we get auto-formatting for template strings, which is pretty magic, and really nice.